.cbmc-batch/jobs/aws_cryptosdk_rsa_encrypt/aws_cryptosdk_rsa_encrypt_harness.c

```diff
@@ -39,6 +39,9 @@ void aws_cryptosdk_rsa_encrypt_harness() {
     struct store_byte_from_buffer old_byte_from_plain;
     save_byte_from_array(plain.ptr, plain.len, &old_byte_from_plain);
 
+    /*initialize a nondeterministic but fixed max encryption size between 0 and INT_MAX */
+    initialize_max_encryption_size();
```

Comment:
would it make more sense for it to take a parameter with the size to init to?

Reply:
I could change the function to do this. I believe these functions were written to be analogous to initialize_max_signature_size() and initialize_max_derivation_size(), so to keep everything consistent I should change the signature of those as wells. I don't this that will effect too much, but might make sense to do that in a separate PR.

Reply:
Makes sense, I'm in favor of handling that separately for now. Looks good to me!

Reply:
https://github.com/aws/aws-encryption-sdk-c/issues/590 to track this